alexandria/core/api.py

```diff
@@ -35,6 +37,53 @@ def verify_signed_components(
     )
 
 
+def copy_document(
```

Comment:
It would make sense to define this as atomic using `@transaction.atomic`

Reply:
Changed